public/app/components/user/editUser/editUserForm.svc.js

```diff
@@ -111,6 +111,8 @@
           data = { community_id: user.community_id };
           break;
         case 'birthday':
+          //user date of birth is empty,it need to be combined from 3 values.
+          user.date_of_birth =  new Date(user.date_of_birth.year,user.date_of_birth.month.value,user.date_of_birth.day);
```

Comment:
```javascript
user.date_of_birth = new Date(user.date_of_birth.year, user.date_of_birth.month.value, user.date_of_birth.day);
```
also, why is `month` the only one with `.value` does it write the months name otherwise?